pipeline-steps/bootstrap.yaml

```diff
@@ -43,9 +43,9 @@ steps:
 - task: AzureCLI@1
   displayName: 'Bootstrap'
   inputs:
-    azureSubscription: $(serviceConnection)
+    azureSubscription: ${{ parameters.serviceConnection }}
```

Comment:
why are swapping this?

Reply:
serviceConnection moved from a variable to a parameter